src/styles/battleParty.less

```diff
@@ -0,0 +1,266 @@
+    .clickable {
+        position: absolute;
+        left: 12%;
+        height: 70%;
+        width: 70%;
+        .partyslot {
+            .pokemonName {
+                left: 50%;
+            }
+        }
+    }
+    .content {
+      position: absolute;
+      width: 90%;
+      height: 90%;
+      top: 0px;
+      left: 0px;
+      bottom: 10px;
+      .progress {
+        position: absolute;
+        bottom: 10px;
+        left: 10%;
+        width: 80%;
+        margin: 0px auto 0 auto;
+        span {
+          position: absolute;
+          display: block;
+          width: 100%;
+        }
+      }
+      img {
+        height: 50%;
+        width: 50%;
+        bottom: 20px;
+      }
+    }
+    &:nth-child(5n+0) {
+      img.hatching, .hatching svg,
+      img.hatchingSoon, .hatchingSoon svg {
+        animation-delay: 0.48s;
+      }
+    }
+    &:nth-child(5n+1) {
+      img.hatching, .hatching svg,
+      img.hatchingSoon, .hatchingSoon svg {
+        animation-delay: 0.22s;
+      }
+    }
+    &:nth-child(5n+2) {
+      img.hatching, .hatching svg,
+      img.hatchingSoon, .hatchingSoon svg {
+        animation-delay: 0.35s;
+      }
+    }
+    &:nth-child(5n+3) {
+      img.hatching, .hatching svg,
+      img.hatchingSoon, .hatchingSoon svg {
+        animation-delay: 0.15s;
+      }
+    }
+    img.battlePartyPokemon {
+        width : 100%;
+      animation: wobble-hor-bottom 3s infinite both;
+    }
+    img.hatchingSoon, .hatchingSoon svg {
+      animation: wobble-hor-bottom 5s infinite both;
+    }
+  }
+
+  .battlePartyPokemonSlot:before {
+    content: '';
+    width: 0;
+  }
+
+  .battlePartyPokemonSlot:before {
+    white-space: normal;
+    display: inline-block;
+    vertical-align: middle;
+    max-width: 100%;
+  }
+
+  .battlePartyPokemonSlot:before {
+    padding-top: 100%;
+    /* makes expand to a square */
+  }
+
+  #battlePartyPokemonList {
+    background-color: #aaa;
+  }
+
+  #middle-column {
+    .battlePartyPokemonSlot {
+      flex-basis: 25%;
+      max-width: 25%;
+    }
+    #hatcheryQueue .queuePokemon {
+      width: 11%;
+    }
+  }
+
+  @battlePartyPokemonMainColors:
+    "Bug" #9df68e,
+    "Dark" #3d3d3d,
+    "Dragon" #ff0006,
+    "Electric" #ffe600,
+    "Fairy" #f36ec2,
+    "Fighting" #fbbd47,
+    "Fire" #d94c32,
+    "Flying" #a0c7e8,
+    "Ghost" #dbdbdb,
+    "Grass" #a1f173,
+    "Ground" #ad5840,
+    "Ice" #00aaff,
+    "Normal" #e8e0a0,
+    "Poison" #c183c1,
+    "Psychic" #fa92b2,
+    "Rock" #d2c8ac,
+    "Steel" #b5b4b1,
+    "Water" #379eff;
+
+  @battlePartyPokemonAccentColors:
+    "Bug" #5ab690,
+    "Dark" #111,
+    "Dragon" #111,
+    "Electric" #ccbb00,
+    "Fairy" #a1382b,
+    "Fighting" #b94a00,
+    "Fire" #fafb00,
+    "Flying" #6896c6,
+    "Ghost" #7c7c7c,
+    "Grass" #2f9f59,
+    "Ground" #763900,
+    "Ice" #00ffbe,
+    "Normal" #689050,
+    "Poison" #682a68,
+    "Psychic" #a13959,
+    "Rock" #6d6d6d,
+    "Steel" #7088b5,
+    "Water" #445e9c;
```

Comment:
Are these different from the existing colors we already have?

Reply:
Will remove, was copied from hatchery